README.md

```diff
@@ -25,16 +25,55 @@ git clone https://github.com/aerogear/mobile-developer-console && cd mobile-deve
 4. Log in with user `developer` (developer/123)
 5. Follow the link to mobile-developer-console
 
-### Full experience (Use APB to deploy mobile-developer-console)
+### Full experience
+
+#### Spin up OpenShift cluster locally
+
+**(OpenShift 3.9) Use mobile-core installer**
 
 1. Use [Mobile Core installer](https://github.com/aerogear/mobile-core) to run `oc cluster up` with
 all required configuration to deploy the mobile-developer-console from APB.
 You can follow the docs [here](https://github.com/aerogear/mobile-core)
-2. After your local OpenShift cluster is running, you can navigate to your project in OpenShift console and search for `Mobile Developer Console` in the catalog.
-3. Follow the wizard to provision Mobile Developer Console APB to your project
-4. Repeat steps 2 & 3 for `Mobile CI/CD` APB 
-5. Once both APBs are provisioned (provisioning of `Mobile CI/CD` will take couple of minutes), navigate the link to mobile-developer-console and log in as `developer`
+2. Follow the [steps below](#Provision-Mobile-Services)
+
+**(OpenShift 3.11) Use `oc cluster up` or `minishift`**
+
+:penguin: Linux
+See [OpenShift documentation](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) to learn how to spin up OpenShift 3.11 cluster locally using `oc cluster up` with following components enabled:
+* automation-service-broker
+* service-catalog
+* template-service-broker
```

Comment:
is it possible to just have an example here to show the right `oc cluster up` command to run? This way users don't have to go to the openshift doc.